renovate.json

```diff
@@ -20,30 +20,26 @@
         {
           "groupName": "openstack-k8s-operators",
           "matchPackagePrefixes": ["github.com/openstack-k8s-operators"],
-          "excludePackageNames": ["github.com/openstack-k8s-operators/openstack-baremetal-operator/api"],
-          "schedule": [
-            "every weekend"
-          ]
```

Comment:
Are schedule removals for things other than k8s and controller-runtime intentional?

Reply:
Yes, just switching to a global scheduling config like we use elsewhere [1], since everything was the same for each rule anyhow.

[1] https://github.com/openstack-k8s-operators/mariadb-operator/blob/b4290b2cc183600644fb37e91b5b04669a59f5c9/renovate.json#L14-L45